### DISCUSSION TITLE
Is this a bug in the documentation?

### DISCUSSION THREAD

**Discussion by makeevolution** [sqlalchemy/sqlalchemy#8802] [Usage Questions]
In this part of the documentation https://docs.sqlalchemy.org/en/14/orm/dataclasses.html#orm-imperative-dataclasses, the tables are initialized using ```MetaData()```, instead of ```mapper_registry.metadata```. This causes the mapper in ```start_mappers()``` to not register the table. If I use ```mapper_registry.metadata``` instead, it works. Is this a bug?

**Comment by zzzeek** [comment]
So I dont know what "start_mappers()" is, so the example by itself isn't exactly a "bug", since you can use whatever metadata object you want, but it's likely more typical (such as whatever this "start_mappers()" expects) that it would share the same metadata object that's already on the registry, so we can accept a PR to improve this doc.
